docs/src/modules/components/Notifications.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable react/no-danger, react-hooks/exhaustive-deps */
-import 'isomorphic-fetch';
+import 'cross-fetch';
```

Comment:
Cross fetch seems to use a ponyfill by default, while isomorphic fetch to use a polyfill.